GPU/Common/SoftwareTransformCommon.cpp

```diff
@@ -590,6 +590,8 @@ void SoftwareTransform::BuildDrawingParams(int prim, int vertexCount, u32 vertTy
 		result->drawBuffer = transformedExpanded;
 		result->drawIndexed = true;
 	} else if (prim == GE_PRIM_LINES) {
+		// TODO: What about GE_PRIM_LINE_STRIP?
```

Comment:
Doesn't indexGen always turn them into lines?  I don't think it tries to keep line strip if "pure".

-[Unknown]

Reply:
Mm, I guess it does, yeah.